src/js/config/runtimeConfig.js

```diff
@@ -24,4 +24,15 @@ var runtimeConfig = {
     return "http://mesos-sandbox-ui/?executorId=" + executorId;
   }
 
+  // The generate build a link to diagnose broken deployments
+  // It returns a url on which a GET will be made.
+  // the api calls is expected to return a json of the form:
+  // {
+  //   success: true
+  //   hints: [ String, String, [..] ]
+  // }
+  deploymentDiagnosisUrlGenerate: function() {
+    return null;
```

Comment:
Shouldn't we have a concrete URL here? Is this for another review?
At the moment, I am unable to test the patch end-to-end.

Reply:
that's expected since the url it should point to is internal. So it will be deployed as part of our runtimeConfig (which is not packaged here since we maintain a simili separation between the open source marathon with opt-in features and configuration to make them work in our env)

Reply:
Perfect, thanks.